src/python/pants/notes/2.3.x.md

```diff
@@ -2,6 +2,16 @@
 
 See https://www.pantsbuild.org/v2.3/docs/release-notes-2-3 for an overview of the changes in this release series.
 
+## 2.3.0rc1 (Feb 25, 2021)
+
+### Bug fixes
+
+* Upgrade Pex to 2.1.32 to fix venv shebangs being too long for some Linux systems. (Cherry-pick of #11604) ([#11606](https://github.com/pantsbuild/pants/pull/11606))
+
+### Documentation
+
+* Less chatty plugin resolution logging (Cherry-pick of #11593) ([#11595](https://github.com/pantsbuild/pants/pull/11595))
```

Comment:
Awkward to categorize but it seemed worth noting. There's a tiny perf aspect and then the logging. There was precedent for logging level changes below in `Documentation`.

Reply:
Yeah I think documentation works, or feel free to rename the title to be "Avoid resolving plugins if none specified" and put in Performance. the PR does both things.